### PR TITLE
Add Last Updated field to Traffic Portal

### PR DESCRIPTION
**Pull request by shamrickus**
This PR adds the Last Updated field to every DS form in Traffic Portal (tpv2 already has them).
<hr/>
## Which Traffic Control components are affected by this PR?

- Traffic Portal

## What is the best way to verify this PR?
Verify the last updated field is present on each type of Delivery Service and is correct in Traffic Portal.

## PR submission checklist
- [x] This PR has tests 
- [x] This PR has documentation 
- [x] This PR has a CHANGELOG.md entry 
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://apache.org/security) for details)

### CODE REVIEW

**Comment by rimashah25** [comment]
I see the `Last Updated` field in the form correctly
image
but in the DS table it shows as Invalid Date
image

File: CHANGELOG.md
Comment (mitchell852) [inline CHANGELOG.md:11]: @shamrickus - would you mind adding a link to the issue or PR?

Comment (mitchell852) [reply CHANGELOG.md:11]: also, i can merge this if you fix the conflicts in this file.

**Review by rimashah25 — APPROVED** [review]
Code LGTM. On local I still see the invalid date (even for master branch). So, it seems like a local issue and not related to this PR. 
Need to resolve conflicts before code can be merged.

File: CHANGELOG.md
Comment (mitchell852) [reply CHANGELOG.md:11]: nm, looks like @rimashah25 found a bug

File: CHANGELOG.md
Comment (mitchell852) [inline CHANGELOG.md:51]: looks like a misplaced paren

**Review by ocket8888 — APPROVED** [review]

**Pull request merged**
